src/bit_machine/mod.rs

```diff
@@ -548,20 +569,24 @@ impl SetTracker {
 #[derive(Copy, Clone, Debug)]
 struct NoTracker;
```

Comment:
In 4821b6537c3578714924efb84b36ac83694aa365:

I believe we should make `NoTracker` (and probably `SetTracker`) public now.

Reply:
Fixed